packages/nloge/nloge.0.0.2/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "Non-blocking logger with eio"
+maintainer: ["Satoru Kawahara <s1311350@gmail.com>"]
+authors: ["Satoru Kawahara <s1311350@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/Nymphium/nloge"
+doc: "https://nymphium.github.io/nloge"
+bug-reports: "https://github.com/Nymphium/nloge/issues"
+dev-repo: "git+https://github.com/Nymphium/nloge.git"
+depends: [
+  "dune" {>= "3.7.0"}
+  "eio"
+  "ppx_deriving"
+  "yojson"
```

Comment:
```suggestion
  "yojson" {>= "1.6.0"}
```

This may fix the lower bounds check.

ref:
- https://github.com/ocaml-community/yojson/blob/93ff0388a876992d370345539cf1ff00553b9bbf/lib/type.ml#L3
- https://github.com/ocaml-community/yojson/blob/8129876b3ffe611a9519f1655c2627a24732d58f/lib/type.ml#L3
- it seems that `Yojson.Safe.t` is only available from version 1.6.0

### CI failure excerpt (from [here](https://toxis.caelum.ci.dev/github/ocaml/opam-repository/commit/16602cf16afcc9ff5f6245823e6546c44355c087/variant/compilers,5.0,nloge.0.0.2,lower-bounds))
```
-> installed ppx_deriving.5.2
[ERROR] The compilation of nloge.0.0.2 failed at "dune build -j 31 -p nloge".

#=== ERROR while compiling nloge.0.0.2 ========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/Nymphium/nloge/releases/download/0.0.2/nloge-0.0.2.tbz)
# path                 ~/.opam/5.0/.opam-switch/build/nloge.0.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -j 31 -p nloge
# exit-code            1
# env-file             ~/.opam/log/nloge-7-e582c8.env
# output-file          ~/.opam/log/nloge-7-e582c8.out
### output ###
# (cd _build/default && /home/opam/.opam/5.0/bin/ocamlc.opt -w -40 -alert -unstable -g -bin-annot -I src/.nloge.objs/byte -I /home/opam/.opam/5.0/lib/bigarray-compat -I /home/opam/.opam/5.0/lib/bigstringaf -I /home/opam/.opam/5.0/lib/biniou -I /home/opam/.opam/5.0/lib/bytes -I /home/opam/.opam/5.0/lib/camlp-streams -I /home/opam/.opam/5.0/lib/cstruct -I /home/opam/.opam/5.0/lib/easy-format -I /home/opam/.opam/5.0/lib/eio -I /home/opam/.opam/5.0/lib/eio/core -I /home/opam/.opam/5.0/lib/fmt -I /home/opam/.opam/5.0/lib/hmap -I /home/opam/.opam/5.0/lib/lwt -I /home/opam/.opam/5.0/lib/lwt-dllist -I /home/opam/.opam/5.0/lib/optint -I /home/opam/.opam/5.0/lib/ppx_deriving/runtime -I /home/opam/.opam/5.0/lib/ptime -I /home/opam/.opam/5.0/lib/ptime/clock/os -I /home/opam/.opam/5.0/lib/result -I /home/opam/.opam/5.0/lib/yojson -no-alias-deps -open Nloge__ -o src/.nloge.objs/byte/nloge__Emit.cmo -c -impl src/emit.pp.ml)
# File "src/emit.ml", line 7, characters 26-39:
# 7 | type metadata = (string * Yojson.Safe.t) list
#                               ^^^^^^^^^^^^^
# Error: Unbound type constructor Yojson.Safe.t
```

Reply:
Thank you!